app/components/items/item.tsx

```diff
@@ -0,0 +1,68 @@
+import { getItemData } from "@/utils/utils";
+import { ItemModel } from "@/models/item";
+import ImageViewer from "./clover/image/image";
+import CloverImageViewer from "./clover/image/viewer";
+import AudioViewer from "./clover/audio/viewer";
+import VideoViewer from "./clover/video/viewer";
+import BookViewer from "./clover/book/viewer";
+import PDFViewer from "./clover/pdf/viewer";
+
+const getItem = async (uuid) => {
```

Comment:
Looks like this function isn't being used in this file (and instead it's a prop which is great).